### PR TITLE
docs: readme update

### PR DESCRIPTION
**Pull request by ulilalbab**
Closes 

## Description

Update readme and contribution guidelines
## Current Tasks

- [ ] (Update readme)
- [ ] (Update contribution)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,63 @@
...
-yarn dev
+git clone https://github.com/kawalcovid19/silacak.git
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+Then, install dependencies and fetch data to your local machine:
 
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+```bash
+cd wargabantuwarga.com
```
Comment (resir014) [inline README.md:43]:
```suggestion
cd silacak
```

File: README.md [hunk 2]
```diff
@@ -1,34 +1,63 @@
...
+yarn install
+yarn mirror-box
```
Comment (resir014) [inline README.md:45]:
```suggestion
```

This command is not necessary in Silacak.

File: README.md [hunk 3]
```diff
@@ -1,34 +1,63 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# SILACAK
```
Comment (resir014) [inline README.md:1]:
```suggestion
# Silacak
```

Nitpick: Silacak (sentence case) instead of all uppercase, as dictated by the new design.

File: contributing.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
+# Contributing to SILACAK
```
Comment (resir014) [inline contributing.md:1]:
```suggestion
# Contributing to Silacak
```

Idem.

File: README.md [hunk 5]
```diff
@@ -1,34 +1,63 @@
...
+- Secure
+
+## Developer
+
+Setting Up Silacak in Local.
+
+This is an installation guide for developers before start the project:
+Prerequisite:
+
+1.
```
Comment (resir014) [inline README.md]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -1,34 +1,63 @@
...
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+Then, install dependencies and fetch data to your local machine:
 
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+```bash
+cd wargabantuwarga.com
+yarn install
+yarn mirror-box
+```
```
Comment (adityapurwa) [inline README.md:41]: Is this part relevant to this project? 🤔

File: README.md [hunk 7]
```diff
@@ -1,34 +1,63 @@
...
 
-## Learn More
+## Contributing
 
-To learn more about Next.js, take a look at the following resources:
+See our contribution guidelines [here](contributing.md)
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+## Authors and acknowledgment
```
Comment (resir014) [inline README.md:61]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -1,34 +1,63 @@
...
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+Thanks goes to these wonderful people
```
Comment (resir014) [inline README.md:63]:
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -1,34 +1,63 @@
...
 
-## Deploy on Vercel
+## License
```
Comment (resir014) [inline README.md:65]:
```suggestion
```

File: README.md (hunk 6 above)
Comment (ulilalbab) [reply README.md:46]: I am not sure for this one

File: README.md [hunk 10]
```diff
@@ -1,34 +1,63 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# SILACAK
+
+A progressive web app (PWA) to track COVID19 cases.
+
+## Purpose and Principles
+
+Help field officer (TNI) to easily tracing the cases. According to WHO Tracing ratio should be 1:15.
```
Comment (resir014) [inline README.md:13]:
```suggestion
The main purpose of Silacak is to help our field officers (TNI) easily trace COVID-19 cases in Indonesia. According to WHO Tracing ratio should be 1:15.
```

- Grammar.
- For the second sentence, we need to provide links to any official source to further drive the purpose of this project.

File: README.md [hunk 11]
```diff
@@ -1,34 +1,63 @@
...
+
+### Principles
+
+- Working without reliable connection (even offline mode)
+- New user can input the data easily.
+- Performance
+- Secure
```
Comment (resir014) [inline README.md:20]: For this, we also need to further elaborate on these points, just like what we did on [wargabantuwarga.com](https://github.com/kawalcovid19/wargabantuwarga.com/blob/main/README.md#principles)

We can also copy those principles over here. What do you think, @zainfathoni?

**Review by resir014 — CHANGES_REQUESTED** [review]
First pass of review. Paging @zainfathoni to do his review as well.

File: README.md (hunk 6 above)
Comment (adityapurwa) [reply README.md:46]: I think this part is relevant to our other project, the wargabantuwarga.com

File: README.md (hunk 11 above)
Comment (zainfathoni) [reply README.md:20]: Agreed. However, copying over the content potentially causes them to be out-of-sync when we're updating one of them.
Therefore, please add a link to [the **Principles** section at the original document](https://github.com/kawalcovid19/wargabantuwarga.com#principles) instead.

File: README.md (hunk 6 above)
Comment (zainfathoni) [reply README.md:46]: Exactly, it's irrelevant here. We're neither using the `mirror-box` data nor exporting this site statically.

**Comment by resir014** [comment]
Superseded by #19.

**Comment by zainfathoni** [comment]
Thanks for the attempt @ulilalbab, you can pick some other issues that we have defined in this project. 😁 🙏
